packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -333,6 +333,11 @@ class WebAssetServer implements AssetReader {
   // handle requests for JavaScript source, dart sources maps, or asset files.
   @visibleForTesting
   Future<shelf.Response> handleRequest(shelf.Request request) async {
+    if (request.method == 'POST') {
+      // Assets are served via GET only.
+      return shelf.Response.notFound('');
+    }
```

Comment:
You may need to do the same in `ReleaseAssetServer.handle` but I'm not 100% sure. I'll let @jonahwilliams confirm.

Reply:
Done.